spec/cassandra/cluster/schema/fetchers/2.2.1-data.json

```diff
@@ -3035,6 +3035,17 @@
       "index_type": null,
       "type": "regular",
       "validator": "org.apache.cassandra.db.marshal.Int32Type"
+    },
+    {
+      "keyspace_name": "simplex",
+      "columnfamily_name": "t3",
+      "column_name": "f3",
+      "component_index": 0,
+      "index_name": null,
+      "index_options": "null",
+      "index_type": null,
+      "type": "regular",
+      "validator": "org.apache.cassandra.db.marshal.ListType(org.apache.cassandra.db.marshal.FrozenType(org.apache.cassandra.db.marshal.Int32Type))"
```

Comment:
This looks backwards. Shouldn't it be `org.apache.cassandra.db.marshal.FrozenType(org.apache.cassandra.db.marshal.ListType(org.apache.cassandra.db.marshal.Int32Type))`?

Reply:
Nope, it should be e.g. exactly this: `list<frozen <list<int>>>`.

See https://docs.datastax.com/en/cql/3.3/cql/cql_reference/collection_type_r.html

Reply:
But that leads to this validator in C* (when I try with C* 2.2.9):
`org.apache.cassandra.db.marshal.ListType(org.apache.cassandra.db.marshal.FrozenType(org.apache.cassandra.db.marshal.ListType(org.apache.cassandra.db.marshal.Int32Type)))`

The validator value you have above puts FrozenType around Int32Type, which I think is illegal. You can't freeze a primitive, only a collection.

Reply:
Just confirmed that 2.2.1 also produces the same validator string.